src/main/java/net/mcreator/ui/workspace/breadcrumb/WorkspaceFolderBreadcrumb.java

```diff
@@ -105,20 +127,20 @@ public void reloadPath(FolderElement file, Class<? extends IElement> childElemen
 						}
 
 						for (IElement file : files) {
-							JMenuItem menuItem = new JMenuItem("<html>&nbsp;" + file.getName());
+							JMenuItem menuItem;
+							if (currentLevel < path.size() - 1 && path.get(currentLevel + 1).equals(file))

```

Comment:
If you are to make another attempt at this PR, please also avoid such changes that are not really directly related to the goal of this PR and make the PR more complex when not needed

Reply:
This is fixed by the invisible scroll bar solution you suggested.